packages/build/src/log/messages/compatibility.js

```diff
@@ -146,6 +156,37 @@ export const logIncompatiblePlugins = function (logs, pluginsOptions) {
   logWarningArray(logs, incompatiblePlugins)
 }
 
+// TODO: This is temporary until we auto-install the Next runtime always for supported Next.js versions.
+const nextPluginVersionChecker = function (major, minor, patch) {
+  // maybe there's a semver package to check this
+
+  // version >= 4.0.0 && < 4.26.0
+  return (major === 4 && minor >= 0 && minor < 26) || (minor === 26 && patch === 0)
+}
+
+/**
+ * Throws an error if the Next runtime is >= 4.0.0 || < 4.26.0, otherwise returns.
+ *
+ * @param {object[]} outdatedPlugins
+ *
+ * @throws Error
+ */
+const throwIfOutdatedNextRuntime = function (outdatedPlugins) {
```

Comment:
Even though this is just Next for now, it would be better if this had a generic name

Reply:
Sounds good. I was just being explicit as I thought this was a temporary one off.